.github/workflows/setup-and-test.yml

```diff
@@ -8,11 +8,14 @@ on:
     branches: [ '**' ]
   merge_group:
     branches: [ '**' ]
+  workflow_dispatch:
```

Comment:
Is this a placeholder?

Reply:
no! the presence of `workflow_dispatch:` without any arguments simply makes it possible to run manually from the web UI.